### ISSUE TITLE
Focus & Tabbing fixes needed

### ISSUE THREAD

**Issue by geospatialem** [MetropolitanCouncil/MME#9]
Thank you for the opportunity to comment and test! **_Fabulous_** work on the MME and thank you for working on accessibility!!  

The comments below were obtained through testing in the Chrome browser with the WAVE toolbar and ChromeVox extensions.  

# Focus  
As Kim mentioned, the focus needs additional work, as links seem to be active (using the tab key), but not the text.  

Currently the tab order of the focus looks like so:
![screen capture](https://cloud.githubusercontent.com/assets/5023024/23436149/12eccaa8-fdcf-11e6-9a8b-952b5a7f0ad0.png)

# Tabbing  
While a user can click on a section heading (_e.g. Section 1: Overview_), sub-heading (_e.g. Originator_), and value (_e.g. U.S. Department of Commerce_), users cannot access the information without clicking on it. Ultimately this might mean a screen reader would be unable to pick up any of that information without the user clicking it first.

Similar to the sections, sub-headings, and value text going undetected, the same holds true for the header including: the title (_e.g. ZIP Code Tabulation Areas_), update date (_e.g. This page last updated…_), and how the metadata was created (_e.g. Metadata created using…_).  

Lastly, any sections without information should still have a link placeholder to let the screen reader know there is a section there, but there’s no information. Currently the item is skipped over if there is no information listed in the section, so a user wouldn't be able to find the section (_e.g. Data Organization_).
